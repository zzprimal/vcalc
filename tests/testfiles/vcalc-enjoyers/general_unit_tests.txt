int a = 1;
int b = 2;
if (b - a)
    print(b - a);
    loop (b - a)
        print(b);
        print(a);
        a = a + 1;
    pool;
    a = 4*(3 + 1 > 0) /  2 + 5;
    if (a - b)
        loop (a - b)
            print(b);
            print(a);
            a = a - 1;
        pool; 
    fi;
fi;
vector e = 1..2;
vector f = [i in 1..5 | i];
vector g = [i in 1..5 & i > 2];
print(e + f - g);
print(e + f * g);
print(e + f / g);
print(e * f == g);
print(e * f - g);
print(e > f / g);
//CHECK_FILE:./general_unit_tests.out