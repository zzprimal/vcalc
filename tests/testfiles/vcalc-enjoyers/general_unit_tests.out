1
2
1
2
7
2
6
2
5
2
4
2
3
[-1 0 -2 4 5]
[4 10 15 0 0]
[1 2 0 4 5]
[0 1 0 1 1]
[-2 0 -5 0 0]
[1 1 0 0 0]
